src/main/scala/nl/knaw/dans/easy/stage/Settings.scala

```diff
@@ -39,9 +39,9 @@ case class Settings(ownerId: String,
 object Settings {
 
   /** backward compatible call for EasyIngestFlow */
-  def apply(ownerId: String,
+  def apply(
```

Comment:
Yes I know since it is already described in jira EASY-1055. The easy-ingest-flow and easy-ingest-dispatcher also need to refactor.